docs/source/chatops/chatops.rst

```diff
@@ -76,9 +76,9 @@ StackStorm installation, you will need to set the following environment
 variables:
 
 -  ``ST2_API`` - FQDN + port to StackStorm endpoint. Typically:
-   ``http://<host>:9101``
+   ``https://<host>/api``
```

Comment:
this will likely merge-conflict with https://github.com/StackStorm/st2/pull/2277; 
I'll still merge 2277 first and we figure this out Mon.